### ISSUE TITLE
Trees with root disjunctive node when imported become with conjunctive operator

### ISSUE THREAD

**Issue by vilena** [tahti/ADTool2#5]
Example tree:
<?xml version='1.0'?>
<adtree>
	<node refinement="disjunctive">
		<label>root</label>
		<node refinement="conjunctive">
			<label>N_1</label>
		</node>
		<node refinement="conjunctive">
			<label>N_2</label>
		</node>
	</node>
</adtree>

When I try to import this tree from xml, it changes the root operator to AND instead of OR

**Comment by vilena** [comment]
```
<?xml version='1.0'?>
<adtree>
	<node refinement="disjunctive">
		<label>root</label>
		<node refinement="conjunctive">
			<label>N_1</label>
		</node>
		<node refinement="conjunctive">
			<label>N_2</label>
		</node>
	</node>
</adtree>
```

**Comment by tahti** [comment]
Issue is fixed in version 2.2.0 of ADTool.
